R/module_teal.R

```diff
@@ -112,6 +112,7 @@ ui_teal <- function(id,
     title = title,
     theme = get_teal_bs_theme(),
     include_teal_css_js(),
+    title,
```

Comment:
I don't understand why we need this line since we already passed the `title` object (result of `build_app_title`) in `title=` argument.

I removed this line locally and still get the same behavior.
Would you please clarify?

Reply:
Yeah you're right. One of the `title` can be removed. Right now it adds two title tags in the head. I'll keep the title argument as it's more explicit.